### ISSUE TITLE
feat(ci): build with earlier version of GLIBC

### ISSUE THREAD

**Issue by fuyufjh** [risingwavelabs/risingwave#4556]
Got errors when I tried to run RisingWave on my EC2 machine.

```
[ec2-user@ip-172-31-11-6 risingwave]$ ./risingwave
./risingwave: /lib64/libm.so.6: version `GLIBC_2.27' not found (required by ./risingwave)
./risingwave: /lib64/libm.so.6: version `GLIBC_2.29' not found (required by ./risingwave)
./risingwave: /lib64/libc.so.6: version `GLIBC_2.29' not found (required by ./risingwave)
./risingwave: /lib64/libc.so.6: version `GLIBC_2.28' not found (required by ./risingwave)
./risingwave: /lib64/libc.so.6: version `GLIBC_2.34' not found (required by ./risingwave)
./risingwave: /lib64/libc.so.6: version `GLIBC_2.32' not found (required by ./risingwave)
./risingwave: /lib64/libc.so.6: version `GLIBC_2.33' not found (required by ./risingwave)
```

The OS is the latest version of Amazon Linux, which is the default option recommended by AWS. The GLIBC version shipped with OS is 2.26.

According to [this article](https://kobzol.github.io/rust/ci/2021/05/07/building-rust-binaries-in-ci-that-work-with-older-glibc.html), the solution is to build binaries on a earlier Linux distribution (therefore with earlier version GLIBC)

```
container: quay.io/pypa/manylinux2014_x86_64
```

We may take a try.

**Comment by skyzh** [comment]
Or we can use musl instead of glibc?

**Comment by BugenZhao** [comment]
> Or we can use musl instead of glibc?

How does musl performs compared to libc now?

**Comment by skyzh** [comment]
No idea :(

**Comment by fuyufjh** [comment]
I also had some concerns about the performance of musl. Earlier version GLIBC seems a better choice and it's [adopted by Python officially](https://peps.python.org/pep-0599/).

**Comment by gitccl** [comment]
Same problem on ubuntu20.04.
```
./risingwave: /lib64/libc.so.6: version `GLIBC_2.34' not found (required by ./risingwave)
./risingwave: /lib64/libc.so.6: version `GLIBC_2.32' not found (required by ./risingwave)
./risingwave: /lib64/libc.so.6: version `GLIBC_2.33' not found (required by ./risingwave)
```

**Comment by fuyufjh** [comment]
@huangjw806 any updates?

**Comment by huangjw806** [comment]
It has been solved, and the packages after v0.1.11 should be able to run normally on the above machines.
